tests/snapshots/examples__build_minecraft_1_18__0.data_pack/pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
   "pack": {
-    "pack_format": 9,
+    "pack_format": 8,
```

Comment:
This illustrates that this PR is a breaking change. Specifying `minecraft: 1.18` in the config now specifies exactly `1.18`, not the last `1.18.x`. I know we discussed this a couple months ago in a google docs, but we'll need to be very careful merging this if people rely on this config.